### DISCUSSION TITLE
Option to hide menu

### DISCUSSION THREAD

**Discussion by voidray** [smplayer-dev/smplayer#361] [Ideas]
Currently it is possible to hide the Main Toolbar. It is also hidden when choosing Mini GUI.
I didn't find a way to also hide the Menu (Open, Play, Video, Audio, ...), so it would be great if an option could be added for that.
The menu is also available using right click on the content area, so to me it is not necessary.

**Comment by smplayer-dev** [comment]
It's possible to hide the menu with Ctrl+C.

**Reply by voidray** [reply]
I only want to hide the menu bar, Ctrl+C also hides the control bar.

**Reply by BaconCatBug** [reply]
Posting as I was googling this, I too would love to be able to hide just the top portion. I run SMplayer without any window decorations maximised (so it takes up the whole screen minus the bottom taskbar). I know it's only a tiny handful of pixels at the top, but it would be nice to be able to hide just the top part. Would it be possible to make Control+C cycle between Hide All > Hide Top > Hide Bottom > Show all?

**Comment by JadeVane** [comment]
Want it too.

**Comment by BaconCatBug** [comment]
Any chance this could be re-looked at? It's the one UI element that I can't really hide independent of the bottom control bar, I've resorted to using raw MPV in order to get rid of it.

**Comment by XinHong** [comment]
Yes please vote +1

**Comment by manikislammahi** [comment]
Still it's not possible in 2023?

**Comment by smplayer-dev** [comment]
I'll try to implement it someday, but it's not a priority. Besides, it's not as easy to implement as it may seem.
